deployments/admin-api/k8s-redis.yaml

```
@@ -0,0 +1,74 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis-service
+  namespace: redis-work-queue
+spec:
+  selector:
+    app: redis
+  ports:
+  - port: 6379
+    targetPort: 6379
+  clusterIP: None
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)